components/trustyai/trustyai.go

```diff
@@ -21,7 +21,9 @@ import (
 var (
 	ComponentName     = "trustyai"
 	ComponentPathName = "trustyai-service-operator"
-	Path              = deploy.DefaultManifestPath + "/" + ComponentPathName + "/base"
+	PathUpstream      = deploy.DefaultManifestPath + "/" + ComponentPathName + "/overlays/odh"
+	PathDownstream    = deploy.DefaultManifestPath + "/" + ComponentPathName + "/overlays/rhoai"
```

Comment:
Would be great if the paths were unified across the components that this definition could be kept in one place.

Reply:
it is very much up to each component how they would like to have their manifests.
e.g some from DW they only use "rhoai" for different "platform"
in trustyai's case, the componentPathName is not the same as ComponentName.
and some even have a dependentcomponent, which adds more complacity